### DISCUSSION TITLE
Earth radius in the model level specification

### DISCUSSION THREAD

**Discussion by chloe-eagle** [MetOffice/simulation-systems#121] [UM]
Please does anyone know the value that’s used for the earth radius in the model level specification or where can I find out?

**Comment by mcdalvi** [comment]
This is tentatively set in [src/constants/planet_constants_mod.F90](https://code.metoffice.gov.uk/trac/um/browser/main/trunk/src/constants/planet_constants_mod.F90?rev=121916#L233) for 'earth' but can be modified via the _planet_constants_ namelist in the suite/configuration.

**Reply by chloe-eagle** [reply]
Thank you very much
